src/main/java/com/google/cloud/spanner/pgadapter/ProxyServer.java

```diff
@@ -149,15 +153,15 @@ void runServer() throws IOException {
       logger.log(
           Level.INFO,
           "Socket exception on port {0}: {1}. This is normal when the server is stopped.",
-          new Object[] {this.options.getProxyPort(), e});
+          new Object[] {getLocalPort(), e});
```

Comment:
Log the actual port instead of the configured port, as this will be a randomly assigned port if the user has specified port number 0 (zero).